src/main/kotlin/org/malachite/estella/interview/domain/InterviewPayload.kt

```diff
@@ -6,8 +6,9 @@ import java.sql.Timestamp
 import java.time.Instant
 import java.util.*
 
-data class InterviewPayloads(val dateTime: Timestamp = Timestamp.from(Instant.now()),
-                             val minutesLength: Int = 30)
+data class InterviewPayload(val dateTime: Timestamp? = Timestamp.from(Instant.now()),
+                            val minutesLength: Int? = 30)
```

Comment:
Po co nam domyślna wartość skoro jest to nullowalne (lub czemu domyślnie nie jest null)?

Reply:
Nie wiem na ile to jest legit ale Intelij twierdzi że używamy tego tylko w 
``` kotlin
fun createInterview(offer: Offer, application: Application, payload: InterviewPayloads = InterviewPayloads())
```
Która jest nigdzie nie używana więc wydaje mi się że to wszystko jest do wywalenia.